docs.wrm/getting-started.wrm

```diff
@@ -90,14 +90,34 @@ const provider = new ethers.providers.Web3Provider(window.ethereum)
 
 // The Metamask plugin also allows signing transactions to
 // send ether and pay to change state within the blockchain.
-// For this, we need the account signer...
+// For this, you need the account signer...
 const signer = provider.getSigner()
 
+_subsection: Connecting to Ethereum: RPC @<getting-started--connecting-rpc>
```

Comment:
@ricmoo I am not 100% sure where this section came from, I didn't add it. Do I have a slightly out of date branch and you removed it? I might argue some benefit for it to stay in some shape (I already fixed a couple of things below)

Reply:
It looks like I may have had it as a stub that I removed. I'm find adding it back. :)